doc/man/8/ceph.rst

```diff
@@ -1024,6 +1033,16 @@ Usage::
 
 	ceph osd setmaxosd <int[0-]>
 
+Subcommand ``set-require-min-compat-client`` configures the oldest Ceph client
+version allowed to connect to the cluster. Once set, Ceph will prevent you from
+enabling features that will break compatibility with those clients. Please note,
+This subcommand requires that all connected daemons and clients support the
```

Comment:
It might be worth clarifying that this command doesn't actually change which clients can connect.  However, it does prevent you from making any other changes (e.g., crush tunables, or using new features) that would violate the current setting.